packages/fury-adapter-oas3-parser/lib/parser/oas/parseComponentsObject.js

```diff
@@ -41,7 +41,7 @@ function parseComponentsObject(context, element) {
 
   const parseSchemasObject = pipeParseResult(namespace,
     validateIsObject('schemas'),
-    parseObject(context, parseSchemaMember));
+    parseObject(context, name, parseSchemaMember));
```

Comment:
There is still a `validateIsObject` check above because it has a different warning format in regards to the name. I'll adapt this separately, perhaps something like making it so the name may be an array of names:

```js
parseObject(context, [name, 'schemas'], parseSchemaMember)
```

Reply:
I'd use composition to create something like `parsePropertyAsObject` where the validation of the property with the property name would happen. Just quick a thought from the top of my head.

Reply:
Good idea!